src/lib/plugin_apis/crypto.api

```diff
@@ -89,6 +89,19 @@ gboolean bd_crypto_device_is_luks (const gchar *device, GError **error);
  */
 gchar* bd_crypto_luks_uuid (const gchar *device, GError **error);
 
+/**
+ * bd_crypto_luks_metadata_size:
```

Comment:
I would prefer ``bd_crypto_luks_get_metadata_size``, it's a function, after all. I know there's a similar ``bd_crypto_luks_status`` right below, but that's a remnant of the original *blivet.devicelibs* code legacy.

Reply:
Function renamed.